projects/packages/stats-admin/src/class-rest-controller.php

```diff
@@ -403,6 +403,17 @@ public function register_rest_routes() {
 				'permission_callback' => array( $this, 'can_user_view_general_stats_callback' ),
 			)
 		);
+
+		// Purchases endpoint.
+		register_rest_route(
+			static::$namespace,
+			sprintf( '/sites/%d/purchases', Jetpack_Options::get_option( 'id' ) ),
+			array(
+				'methods'             => WP_REST_Server::EDITABLE,
```

Comment:
```suggestion
				'methods'             => WP_REST_Server::READABLE,
```
The API endpoint should work with the `GET` method in my testing.